integration-test.sh

```diff
@@ -467,6 +473,41 @@ function test_docker_dev_image {
     tests_for_all_docker_images
 }
 
+function test_node_management_commands {
+    local cfg
+    local dist
+    local build_type
+    random_configuration cfg
+    random_distribution dist
+    random_build_type build_type
+
+    # for Docker we force the most recent distribution as we don't have Docker images for all versions that are tested
+    if [[ "$build_type" == "docker" ]]; then
+      dist="${DISTRIBUTIONS[${#DISTRIBUTIONS[@]}-1]}"
+    fi
+
+    info "test install [--configuration-name=${cfg}] [--build-type=${build_type}]"
+    kill_rally_processes
+
+    raw_install_id=$(esrally install --quiet --configuration-name="${cfg}" --distribution-version="${dist}" --build-type="${build_type}" --node-name="rally-node-0" --master-nodes="rally-node-0" --network-host="127.0.0.1" --http-port=39200 --seed-hosts="127.0.0.1:39300")
+    install_id=$(echo "${raw_install_id}" | grep installation-id | cut -d '"' -f4)
```

Comment:
Don't want to rely on the presence of `jq` I presume?

Reply:
Yes, this was my intention. My hope is that with https://github.com/elastic/rally/issues/736 we'll extract this in Python anyway.